### DISCUSSION TITLE
blog.concourse-ci.org down?

### DISCUSSION THREAD

**Discussion by ppseafield** [concourse/concourse#8657] [General]
When clicking the link to the blog on concourse-ci.org, I simply see "Site unavailable".

![image](https://user-images.githubusercontent.com/734908/210606686-9ceb5f7d-9093-4f1c-bd68-306d68ab1d7b.png)

**Comment by alistardust** [comment]
Still down, and the lack of responses around here is starting to worry me.

**Comment by ghostsquad** [comment]
any news on this?

**Comment by evanchaoli** [comment]
@xtremerui

**Comment by xtremerui** [comment]
Sorry for late notice. The blog has being shutdown temporarily. We are looking to host the blog somewhere else.

**Comment by xtremerui** [comment]
The Concourse blog is now rehost on github pages. All the posts are restored. We still need to match the theme to concourse-ci.org.

Please let us know if there is any issue. Thank you!
